README.md

```diff
@@ -24,10 +24,11 @@ It is as simple as:
 
 ```rust
 use calamine::{Result, Sheets, RangeDeserializerBuilder};
+use std::io::File;
 
 fn example() -> Result<()> {
     let path = format!("{}/tests/tempurature.xlsx", env!("CARGO_MANIFEST_DIR"));
-    let mut workbook = Sheets::open(path)?;
+    let mut workbook = Sheets::<File>::open(path)?;
```

Comment:
Maybe the explicit `File` type parameter requirement can be avoided somehow.

Reply:
It doesn't really bother me